.gitignore

```diff
@@ -3,3 +3,4 @@ npm-debug.log*
 .nyc_output/
 coverage/
 build/
+.idea/
```

Comment:
No. https://stackoverflow.com/a/7335487/1317451

Reply:
And then, as [I did last time](https://github.com/ePages-de/immutable-api-records/pull/20/commits/58f4ffcc91c490ac04077ea0f43920532aa1523a), please bump the version number in package.json.